teamcity: failed test: TestLogic

The following tests appear to have failed on master (test): TestLogic/fakedist/subquery_correlated, TestLogic/fakedist-metadata/conditional, TestLogic/local-vec-off/truncate/truncate_29010, TestLogic/local/cascade/DeleteSetDefault_Basic1, TestLogic/local/cascade_opt/DeleteCascade_CompositeFKs_MatchSimple, TestLogic/local-mixed-19.1-19.2/srfs/srf_errors, TestLogic/fakedist/crdb_internal/max_retry_counter, TestLogic/fakedist-vec-off/rows_from, TestLogic/fakedist-disk/upsert/visible_returning_columns, TestLogic/local/alter_column_type/BadStringLocale, TestLogic/local-mixed-19.1-19.2/cascade/MatchFullViaCascade, TestLogic/fakedist-metadata/insert/string_bytes_conflicts, TestLogic/fakedist-disk/update/reject_special_funcs_inset, TestLogic/local-vec-off/srfs/srf_accessor, TestLogic/fakedist-metadata/cascade_opt/DefaultSetDefault_Unique, TestLogic/local-mixed-19.1-19.2/schema_change_in_txn-mixed-19.1-19.2/create_as_with_reuse_column_index_name_in_txn, TestLogic/local/crdb_internal/max_retry_counter, TestLogic/fakedist/int_size/issue_32846, TestLogic/fakedist/update/contraint_check_validation_ordering, TestLogic/fakedist-disk/truncate/truncate_30547, TestLogic/fakedist-disk/cascade_opt/DeleteCascade_CompositeFKs_MatchFull, TestLogic/fakedist/create_index, TestLogic/fakedist-vec-off/upsert, TestLogic/fakedist-disk/srfs/expandArray, TestLogic/fakedist-vec-off/storing, TestLogic/local/int_size, TestLogic/local-mixed-19.1-19.2/rename_constraint, TestLogic/fakedist/alter_column_type/BadStringLocale, TestLogic/local/cascade_opt/UpdateSetNull_Basic1, TestLogic/local-vec-off/select_index, TestLogic/fakedist/cascade/UpdateCascade_Multi, TestLogic/fakedist-vec-off/alias_types, TestLogic/fakedist/datetime/regression_36146, TestLogic/fakedist-disk/cascade/DeleteCascade_Restrict, TestLogic/fakedist-vec-off/update, TestLogic/fakedist/cascade_opt/UpdateSetDefault_Unique, TestLogic/local/truncate/truncate_interleave, TestLogic/fakedist-metadata/schema_change_in_txn/create_as_fail_unique_index, TestLogic/fakedist-vec-off/returning, TestLogic/fakedist-metadata/select_index, TestLogic/local-mixed-19.1-19.2/cascade/UpdateCascade_WithChecks, TestLogic/local-mixed-19.1-19.2/cascade_opt/DeleteSetNull_ToUpdateCascade, TestLogic/local-mixed-19.1-19.2/cascade/UpdateCascade_CompositeFKs_MatchSimple, TestLogic/fakedist-vec-off/manual_retry, TestLogic/fakedist/table, TestLogic/local/fk/Composite_Full_Validate_Later, TestLogic/fakedist-disk/subquery, TestLogic/local/cascade/AllCascadingActions, TestLogic/fakedist-metadata/bit/bit_ordering, TestLogic/local/json_builtins, TestLogic/fakedist/cascade/UpdateCascade_CompositeFKs_MatchSimple, TestLogic/fakedist-vec, TestLogic/fakedist-vec-off/tuple/labeled_tuple_column_access, TestLogic/fakedist-metadata/fk, TestLogic/fakedist-vec-off/time, TestLogic/local/fk_opt/20042, TestLogic/fakedist-metadata/cascade_opt/DeleteSetDefault_Basic1_WrongDefault, TestLogic/fakedist-disk/sequences/virtual_sequences, TestLogic/fakedist-metadata/cascade_opt/UpdateSetNull_Basic1, TestLogic/local-vec-off/tuple/literal_labeled_tuple_in_subquery, TestLogic/fakedist-metadata/bit/bit_varying_limited, TestLogic/local-vec-off/upsert/notstrict, TestLogic/local-vec-off/alter_table, TestLogic/fakedist/datetime/regression_35255, TestLogic/local/cascade/UpdateCascade_PrimaryKeys, TestLogic/local/schema_change_in_txn/validate_in_schema_change_txn, TestLogic/fakedist-disk/fk/Composite_Full, TestLogic/fakedist/run_control, TestLogic/fakedist/cascade_opt/DeleteSetDefault_Basic2_WrongDefault, TestLogic/fakedist/cascade/UpdateCascade_Restrict, TestLogic/fakedist/json, TestLogic/local/datetime/regression_36557, TestLogic/local/cascade/DeleteSetDefault_Basic2, TestLogic/local-mixed-19.1-19.2/schema_change_in_txn-mixed-19.1-19.2/create_as_with_add_column_index_in_txn, TestLogic/fakedist/aggregate, TestLogic/fakedist-vec-off/upsert/regression_29497, TestLogic/local/manual_retry/schema_change_with_rollback, TestLogic/fakedist-vec-off/update/regression_35364, TestLogic/local/bit/results, TestLogic/fakedist-vec-off/tuple/labeled_column_access_from_table, TestLogic/local-vec-off/privileges_database, TestLogic/fakedist/cascade/DeleteCascade_Race, TestLogic/local/schema_change_in_txn/create_as_add_multiple_columns, TestLogic/fakedist-vec-off/srfs/correlated_srf, TestLogic/local-mixed-19.1-19.2/srfs/upper, TestLogic/local/cascade/UpdateSetNull_Basic1, TestLogic/local-mixed-19.1-19.2/cascade_opt/UpdateSetNull_ToUpdateCascade, TestLogic/fakedist-disk/select_index_span_ranges, TestLogic/local/srfs/dbviz_example_query, TestLogic/local-mixed-19.1-19.2/cascade_opt/UpdateCascade_Multi, TestLogic/fakedist-disk/srfs/metabase_confluent_example_query, TestLogic/fakedist/fk_opt/Composite_Full_Validate_Constraint_Invalid, TestLogic/fakedist-metadata/errors, TestLogic/fakedist-vec-off/cascade_opt/DeleteCascade_SelfReference, TestLogic/fakedist-metadata/order_by/order_by_index, TestLogic/fakedist-disk/statement_source, TestLogic/fakedist/manual_retry, TestLogic/fakedist-disk/insert/fk_contraint_check_validation_ordering, TestLogic/fakedist-metadata/schema_change_in_txn/35193_computed_column, TestLogic/fakedist/cascade_opt/UpdateCascade_TwoTableLoop

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestLogic).

[#1473393](https://teamcity.cockroachdb.com/viewLog.html?buildId=1473393):


```
TestLogic/fakedist-disk/truncate/truncate_30547
--- FAIL: test/TestLogic/fakedist-disk/truncate/truncate_30547 (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/truncate with config fakedist-disk: 43 tests, 0 failures
--- total progress: 64617 statements/queries



TestLogic/fakedist-metadata/schema_change_in_txn/35193_computed_column
--- FAIL: test/TestLogic/fakedist-metadata/schema_change_in_txn/35193_computed_column (0.000s)
Test ended in panic.




TestLogic/fakedist-disk/update/reject_special_funcs_inset
--- FAIL: test/TestLogic/fakedist-disk/update/reject_special_funcs_inset (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/upsert/regression_29497
--- FAIL: test/TestLogic/fakedist-vec-off/upsert/regression_29497 (0.000s)
Test ended in panic.




TestLogic/local-vec-off/privileges_database
--- FAIL: test/TestLogic/local-vec-off/privileges_database (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/local-vec-off/privileges_database



TestLogic/fakedist-vec-off/storing
--- FAIL: test/TestLogic/fakedist-vec-off/storing (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/storing



TestLogic/local/cascade/UpdateSetNull_Basic1
--- FAIL: test/TestLogic/local/cascade/UpdateSetNull_Basic1 (0.000s)
Test ended in panic.




TestLogic/local/cascade/AllCascadingActions
--- FAIL: test/TestLogic/local/cascade/AllCascadingActions (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/fk_opt: 146 statements/queries



TestLogic/fakedist-metadata/select_index
--- FAIL: test/TestLogic/fakedist-metadata/select_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-metadata/select_index



TestLogic/fakedist-disk/subquery
--- FAIL: test/TestLogic/fakedist-disk/subquery (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-disk/subquery



TestLogic/fakedist/cascade/UpdateCascade_CompositeFKs_MatchSimple
--- FAIL: test/TestLogic/fakedist/cascade/UpdateCascade_CompositeFKs_MatchSimple (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/interleaved_join with config fakedist: 54 tests, 0 failures
=== CONT  TestLogic/fakedist/alter_table
--- progress: testdata/logic_test/alter_column_type: 14 statements/queries



TestLogic/fakedist/fk_opt/Composite_Full_Validate_Constraint_Invalid
--- FAIL: test/TestLogic/fakedist/fk_opt/Composite_Full_Validate_Constraint_Invalid (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/fk: 492 statements/queries
--- progress: testdata/logic_test/fk_opt: 519 statements/queries
--- progress: testdata/logic_test/fk: 504 statements/queries
--- progress: testdata/logic_test/fk_opt: 528 statements/queries
--- progress: testdata/logic_test/fk: 511 statements/queries



TestLogic/local-mixed-19.1-19.2/schema_change_in_txn-mixed-19.1-19.2/create_as_with_add_column_index_in_txn
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/schema_change_in_txn-mixed-19.1-19.2/create_as_with_add_column_index_in_txn (0.000s)
Test ended in panic.




TestLogic/fakedist/cascade_opt/UpdateSetDefault_Unique
--- FAIL: test/TestLogic/fakedist/cascade_opt/UpdateSetDefault_Unique (0.000s)
Test ended in panic.




TestLogic/local/cascade_opt/DeleteCascade_CompositeFKs_MatchSimple
--- FAIL: test/TestLogic/local/cascade_opt/DeleteCascade_CompositeFKs_MatchSimple (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/fk_opt: 211 statements/queries



TestLogic/local-mixed-19.1-19.2/cascade_opt/DeleteSetNull_ToUpdateCascade
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/cascade_opt/DeleteSetNull_ToUpdateCascade (0.000s)
Test ended in panic.




TestLogic/local/cascade/UpdateCascade_PrimaryKeys
--- FAIL: test/TestLogic/local/cascade/UpdateCascade_PrimaryKeys (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/vectorize: 115 statements/queries



TestLogic/local-vec-off/select_index
--- FAIL: test/TestLogic/local-vec-off/select_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/local-vec-off/select_index



TestLogic/fakedist-disk/upsert/visible_returning_columns
--- FAIL: test/TestLogic/fakedist-disk/upsert/visible_returning_columns (0.000s)
Test ended in panic.




TestLogic/local/json_builtins
--- FAIL: test/TestLogic/local/json_builtins (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/local/json_builtins



TestLogic/local/datetime/regression_36557
--- FAIL: test/TestLogic/local/datetime/regression_36557 (0.000s)
Test ended in panic.




TestLogic/fakedist/cascade_opt/UpdateCascade_TwoTableLoop
--- FAIL: test/TestLogic/fakedist/cascade_opt/UpdateCascade_TwoTableLoop (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/explain_analyze with config fakedist: 20 tests, 0 failures
--- progress: testdata/logic_test/fk: 23 statements/queries
--- total progress: 37415 statements/queries
=== CONT  TestLogic/fakedist/distinct_on



TestLogic/local/manual_retry/schema_change_with_rollback
--- FAIL: test/TestLogic/local/manual_retry/schema_change_with_rollback (0.000s)
Test ended in panic.




TestLogic/local/fk/Composite_Full_Validate_Later
--- FAIL: test/TestLogic/local/fk/Composite_Full_Validate_Later (0.000s)
Test ended in panic.




TestLogic/local-mixed-19.1-19.2/schema_change_in_txn-mixed-19.1-19.2/create_as_with_reuse_column_index_name_in_txn
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/schema_change_in_txn-mixed-19.1-19.2/create_as_with_reuse_column_index_name_in_txn (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/table with config local-mixed-19.1-19.2: 67 tests, 0 failures
=== CONT  TestLogic/local-mixed-19.1-19.2/srfs



TestLogic/local-mixed-19.1-19.2/cascade_opt/UpdateCascade_Multi
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/cascade_opt/UpdateCascade_Multi (0.000s)
Test ended in panic.




TestLogic/local/alter_column_type/BadStringLocale
--- FAIL: test/TestLogic/local/alter_column_type/BadStringLocale (0.000s)
Test ended in panic.

------- Stdout: -------
=== CONT  TestLogic/local/zero



TestLogic/fakedist/crdb_internal/max_retry_counter
--- FAIL: test/TestLogic/fakedist/crdb_internal/max_retry_counter (0.000s)
Test ended in panic.

------- Stdout: -------
=== CONT  TestLogic/fakedist/cascade_opt
=== CONT  TestLogic/fakedist/cascade
--- progress: testdata/logic_test/schema_change_in_txn: 250 statements/queries
--- done: testdata/logic_test/collatedstring with config fakedist: 67 tests, 0 failures
--- total progress: 33738 statements/queries
=== CONT  TestLogic/fakedist/bytes
--- progress: testdata/logic_test/privilege_builtins: 75 statements/queries



TestLogic/local/schema_change_in_txn/create_as_add_multiple_columns
--- FAIL: test/TestLogic/local/schema_change_in_txn/create_as_add_multiple_columns (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/rename_sequence with config local: 8 tests, 0 failures



TestLogic/local-mixed-19.1-19.2/cascade/UpdateCascade_WithChecks
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/cascade/UpdateCascade_WithChecks (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/insert/string_bytes_conflicts
--- FAIL: test/TestLogic/fakedist-metadata/insert/string_bytes_conflicts (0.000s)
Test ended in panic.




TestLogic/local/cascade_opt/UpdateSetNull_Basic1
--- FAIL: test/TestLogic/local/cascade_opt/UpdateSetNull_Basic1 (0.000s)
Test ended in panic.




TestLogic/local-vec-off/truncate/truncate_29010
--- FAIL: test/TestLogic/local-vec-off/truncate/truncate_29010 (0.000s)
Test ended in panic.




TestLogic/local/srfs/dbviz_example_query
--- FAIL: test/TestLogic/local/srfs/dbviz_example_query (0.000s)
Test ended in panic.

------- Stdout: -------
=== CONT  TestLogic/local/select_table_alias
=== CONT  TestLogic/local/select_search_path



TestLogic/local-mixed-19.1-19.2/srfs/upper
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/srfs/upper (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/sqlsmith with config local-mixed-19.1-19.2: 10 tests, 0 failures



TestLogic/local/cascade/DeleteSetDefault_Basic2
--- FAIL: test/TestLogic/local/cascade/DeleteSetDefault_Basic2 (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/srfs/correlated_srf
--- FAIL: test/TestLogic/fakedist-vec-off/srfs/correlated_srf (0.000s)
Test ended in panic.




TestLogic/fakedist/alter_column_type/BadStringLocale
--- FAIL: test/TestLogic/fakedist/alter_column_type/BadStringLocale (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/schema_change_in_txn with config fakedist: 462 tests, 0 failures



TestLogic/local-mixed-19.1-19.2/cascade_opt/UpdateSetNull_ToUpdateCascade
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/cascade_opt/UpdateSetNull_ToUpdateCascade (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/tuple/labeled_tuple_column_access
--- FAIL: test/TestLogic/fakedist-vec-off/tuple/labeled_tuple_column_access (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/manual_retry
--- FAIL: test/TestLogic/fakedist-vec-off/manual_retry (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/manual_retry



TestLogic/fakedist-vec-off/tuple/labeled_column_access_from_table
--- FAIL: test/TestLogic/fakedist-vec-off/tuple/labeled_column_access_from_table (0.000s)
Test ended in panic.




TestLogic/fakedist-disk/srfs/expandArray
--- FAIL: test/TestLogic/fakedist-disk/srfs/expandArray (0.000s)
Test ended in panic.




TestLogic/fakedist/run_control
--- FAIL: test/TestLogic/fakedist/run_control (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/run_control



TestLogic/local/crdb_internal/max_retry_counter
--- FAIL: test/TestLogic/local/crdb_internal/max_retry_counter (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/fk_opt: 116 statements/queries
--- progress: testdata/logic_test/fk: 98 statements/queries
--- done: testdata/logic_test/create_as with config local: 58 tests, 0 failures
=== CONT  TestLogic/local/collatedstring_nullinindex
--- done: testdata/logic_test/collatedstring_uniqueindex1 with config local: 14 tests, 0 failures
--- done: testdata/logic_test/collatedstring_uniqueindex2 with config local: 14 tests, 0 failures
=== CONT  TestLogic/local/collatedstring_normalization
=== CONT  TestLogic/local/collatedstring_index2
--- progress: testdata/logic_test/create_index: 24 statements/queries
--- done: testdata/logic_test/create_index with config local: 25 tests, 0 failures
--- total progress: 7074 statements/queries
=== CONT  TestLogic/local/collatedstring_index1
--- done: testdata/logic_test/crdb_internal with config local: 101 tests, 0 failures
=== CONT  TestLogic/local/collatedstring_constraint
--- done: testdata/logic_test/collatedstring_normalization with config local: 6 tests, 0 failures
=== CONT  TestLogic/local/collatedstring
--- progress: testdata/logic_test/computed: 82 statements/queries
--- done: testdata/logic_test/collatedstring_nullinindex with config local: 6 tests, 0 failures
=== CONT  TestLogic/local/check_constraints
--- done: testdata/logic_test/collatedstring_index2 with config local: 19 tests, 0 failures
--- progress: testdata/logic_test/fk_opt: 130 statements/queries
=== CONT  TestLogic/local/ccl
--- done: testdata/logic_test/collatedstring_constraint with config local: 18 tests, 0 failures
=== CONT  TestLogic/local/case_sensitive_names
--- progress: testdata/logic_test/fk: 123 statements/queries
--- done: testdata/logic_test/collatedstring_index1 with config local: 19 tests, 0 failures
=== CONT  TestLogic/local/cascade_opt
--- done: testdata/logic_test/collatedstring with config local: 67 tests, 0 failures
--- total progress: 7310 statements/queries
=== CONT  TestLogic/local/cascade
--- done: testdata/logic_test/ccl with config local: 9 tests, 0 failures
=== CONT  TestLogic/local/bytes



TestLogic/fakedist/manual_retry
--- FAIL: test/TestLogic/fakedist/manual_retry (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/manual_retry



TestLogic/local-mixed-19.1-19.2/srfs/srf_errors
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/srfs/srf_errors (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/schema_change_in_txn/create_as_fail_unique_index
--- FAIL: test/TestLogic/fakedist-metadata/schema_change_in_txn/create_as_fail_unique_index (0.000s)
Test ended in panic.




TestLogic/fakedist-disk/cascade_opt/DeleteCascade_CompositeFKs_MatchFull
--- FAIL: test/TestLogic/fakedist-disk/cascade_opt/DeleteCascade_CompositeFKs_MatchFull (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/alter_table: 14 statements/queries



TestLogic/local-vec-off/upsert/notstrict
--- FAIL: test/TestLogic/local-vec-off/upsert/notstrict (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/cascade_opt/DefaultSetDefault_Unique
--- FAIL: test/TestLogic/fakedist-metadata/cascade_opt/DefaultSetDefault_Unique (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/returning
--- FAIL: test/TestLogic/fakedist-vec-off/returning (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/returning



TestLogic/local-mixed-19.1-19.2/cascade/UpdateCascade_CompositeFKs_MatchSimple
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/cascade/UpdateCascade_CompositeFKs_MatchSimple (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/bit/bit_varying_limited
--- FAIL: test/TestLogic/fakedist-metadata/bit/bit_varying_limited (0.000s)
Test ended in panic.




TestLogic/local/fk_opt/20042
--- FAIL: test/TestLogic/local/fk_opt/20042 (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/cascade_opt/DeleteSetDefault_Basic1_WrongDefault
--- FAIL: test/TestLogic/fakedist-metadata/cascade_opt/DeleteSetDefault_Basic1_WrongDefault (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/alias_types
--- FAIL: test/TestLogic/fakedist-vec-off/alias_types (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/alias_types



TestLogic/fakedist-disk/fk/Composite_Full
--- FAIL: test/TestLogic/fakedist-disk/fk/Composite_Full (0.000s)
Test ended in panic.




TestLogic/fakedist-disk/cascade/DeleteCascade_Restrict
--- FAIL: test/TestLogic/fakedist-disk/cascade/DeleteCascade_Restrict (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/array: 138 statements/queries
--- progress: testdata/logic_test/builtin_function: 396 statements/queries
--- progress: testdata/logic_test/fk: 223 statements/queries



TestLogic/fakedist/create_index
--- FAIL: test/TestLogic/fakedist/create_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/create_index



TestLogic/fakedist-metadata/fk
--- FAIL: test/TestLogic/fakedist-metadata/fk (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-metadata/fk



TestLogic/fakedist/datetime/regression_36146
--- FAIL: test/TestLogic/fakedist/datetime/regression_36146 (0.000s)
Test ended in panic.




TestLogic/local/cascade/DeleteSetDefault_Basic1
--- FAIL: test/TestLogic/local/cascade/DeleteSetDefault_Basic1 (0.000s)
Test ended in panic.




TestLogic/fakedist-vec
--- FAIL: test/TestLogic/fakedist-vec (0.000s)
Test ended in panic.




TestLogic/fakedist/cascade/DeleteCascade_Race
--- FAIL: test/TestLogic/fakedist/cascade/DeleteCascade_Race (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/cascade_opt: 121 statements/queries



TestLogic/fakedist/datetime/regression_35255
--- FAIL: test/TestLogic/fakedist/datetime/regression_35255 (0.000s)
Test ended in panic.




TestLogic/local-vec-off/srfs/srf_accessor
--- FAIL: test/TestLogic/local-vec-off/srfs/srf_accessor (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/errors
--- FAIL: test/TestLogic/fakedist-metadata/errors (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-metadata/errors



TestLogic/fakedist-metadata/order_by/order_by_index
--- FAIL: test/TestLogic/fakedist-metadata/order_by/order_by_index (0.000s)
Test ended in panic.




TestLogic/local-vec-off/tuple/literal_labeled_tuple_in_subquery
--- FAIL: test/TestLogic/local-vec-off/tuple/literal_labeled_tuple_in_subquery (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/update/regression_35364
--- FAIL: test/TestLogic/fakedist-vec-off/update/regression_35364 (0.000s)
Test ended in panic.

------- Stdout: -------
=== CONT  TestLogic/fakedist-vec-off/truncate
--- done: testdata/logic_test/update with config fakedist-vec-off: 119 tests, 0 failures
=== CONT  TestLogic/fakedist-vec-off/select_for_update
--- done: testdata/logic_test/upsert with config fakedist-vec-off: 212 tests, 0 failures
--- progress: testdata/logic_test/select_index: 19 statements/queries
--- total progress: 44530 statements/queries
=== CONT  TestLogic/fakedist-vec-off/timestamp
--- progress: testdata/logic_test/alter_table: 102 statements/queries
--- progress: testdata/logic_test/cascade: 198 statements/queries



TestLogic/fakedist/table
--- FAIL: test/TestLogic/fakedist/table (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/table



TestLogic/fakedist/update/contraint_check_validation_ordering
--- FAIL: test/TestLogic/fakedist/update/contraint_check_validation_ordering (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/cascade_opt/UpdateSetNull_Basic1
--- FAIL: test/TestLogic/fakedist-metadata/cascade_opt/UpdateSetNull_Basic1 (0.000s)
Test ended in panic.




TestLogic/fakedist-metadata/bit/bit_ordering
--- FAIL: test/TestLogic/fakedist-metadata/bit/bit_ordering (0.000s)
Test ended in panic.

------- Stdout: -------
--- total progress: 49375 statements/queries
=== CONT  TestLogic/fakedist-metadata/alter_sequence
--- done: testdata/logic_test/computed with config fakedist-metadata: 159 tests, 0 failures
=== CONT  TestLogic/fakedist-metadata/alter_column_type



TestLogic/local-mixed-19.1-19.2/cascade/MatchFullViaCascade
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/cascade/MatchFullViaCascade (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/time
--- FAIL: test/TestLogic/fakedist-vec-off/time (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/time



TestLogic/fakedist-disk/srfs/metabase_confluent_example_query
--- FAIL: test/TestLogic/fakedist-disk/srfs/metabase_confluent_example_query (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/collatedstring with config fakedist-disk: 67 tests, 0 failures



TestLogic/fakedist-vec-off/cascade_opt/DeleteCascade_SelfReference
--- FAIL: test/TestLogic/fakedist-vec-off/cascade_opt/DeleteCascade_SelfReference (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/family: 28 statements/queries



TestLogic/local/int_size
--- FAIL: test/TestLogic/local/int_size (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/local/int_size



TestLogic/local/bit/results
--- FAIL: test/TestLogic/local/bit/results (0.000s)
Test ended in panic.




TestLogic/local-vec-off/alter_table
--- FAIL: test/TestLogic/local-vec-off/alter_table (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/local-vec-off/alter_table



TestLogic/fakedist/cascade/UpdateCascade_Restrict
--- FAIL: test/TestLogic/fakedist/cascade/UpdateCascade_Restrict (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/alter_sequence with config fakedist: 5 tests, 0 failures
--- progress: testdata/logic_test/interleaved: 42 statements/queries
=== CONT  TestLogic/fakedist/inet
--- done: testdata/logic_test/errors with config fakedist: 7 tests, 0 failures
=== CONT  TestLogic/fakedist/function_lookup
--- progress: testdata/logic_test/array: 142 statements/queries
--- progress: testdata/logic_test/cascade_opt: 189 statements/queries



TestLogic/fakedist/cascade/UpdateCascade_Multi
--- FAIL: test/TestLogic/fakedist/cascade/UpdateCascade_Multi (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/fk_opt: 62 statements/queries
--- done: testdata/logic_test/distsql_expr with config fakedist: 3 tests, 0 failures
--- done: testdata/logic_test/distsql_srfs with config fakedist: 7 tests, 0 failures
--- total progress: 37589 statements/queries
=== CONT  TestLogic/fakedist/aggregate-opt
=== CONT  TestLogic/fakedist/alias_types



TestLogic/fakedist-disk/insert/fk_contraint_check_validation_ordering
--- FAIL: test/TestLogic/fakedist-disk/insert/fk_contraint_check_validation_ordering (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/rows_from
--- FAIL: test/TestLogic/fakedist-vec-off/rows_from (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/rows_from



TestLogic/local/truncate/truncate_interleave
--- FAIL: test/TestLogic/local/truncate/truncate_interleave (0.000s)
Test ended in panic.




TestLogic/fakedist/int_size/issue_32846
--- FAIL: test/TestLogic/fakedist/int_size/issue_32846 (0.000s)
Test ended in panic.




TestLogic/fakedist/json
--- FAIL: test/TestLogic/fakedist/json (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/json



TestLogic/fakedist/cascade_opt/DeleteSetDefault_Basic2_WrongDefault
--- FAIL: test/TestLogic/fakedist/cascade_opt/DeleteSetDefault_Basic2_WrongDefault (0.000s)
Test ended in panic.




TestLogic/fakedist-vec-off/upsert
--- FAIL: test/TestLogic/fakedist-vec-off/upsert (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/upsert



TestLogic/fakedist-metadata/conditional
--- FAIL: test/TestLogic/fakedist-metadata/conditional (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-metadata/conditional



TestLogic/fakedist-vec-off/update
--- FAIL: test/TestLogic/fakedist-vec-off/update (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-vec-off/update



TestLogic/fakedist/aggregate
--- FAIL: test/TestLogic/fakedist/aggregate (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/aggregate



TestLogic/local-mixed-19.1-19.2/rename_constraint
--- FAIL: test/TestLogic/local-mixed-19.1-19.2/rename_constraint (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/local-mixed-19.1-19.2/rename_constraint



TestLogic/fakedist-disk/select_index_span_ranges
--- FAIL: test/TestLogic/fakedist-disk/select_index_span_ranges (0.000s)
Test ended in panic.

------- Stdout: -------
Setting kv batch size 10
--- done: testdata/logic_test/select_index_span_ranges with config fakedist-disk: 9 tests, 0 failures



TestLogic/fakedist/subquery_correlated
--- FAIL: test/TestLogic/fakedist/subquery_correlated (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist/subquery_correlated



TestLogic/local/schema_change_in_txn/validate_in_schema_change_txn
--- FAIL: test/TestLogic/local/schema_change_in_txn/validate_in_schema_change_txn (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/postgres_jsonb with config local: 87 tests, 0 failures
--- total progress: 2798 statements/queries
=== CONT  TestLogic/local/pg_catalog
--- progress: testdata/logic_test/privileges_table: 81 statements/queries



TestLogic/fakedist-disk/statement_source
--- FAIL: test/TestLogic/fakedist-disk/statement_source (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/fakedist-disk/statement_source



TestLogic/fakedist-disk/sequences/virtual_sequences
--- FAIL: test/TestLogic/fakedist-disk/sequences/virtual_sequences (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/cascade_opt: 277 statements/queries




```

Please assign, take a look and update the issue accordingly.

Seems to be a timeout:

```
panic: test timed out after 12m0s

goroutine 4391789 [running]:
testing.(*M).startAlarm.func1()
    /usr/local/go/src/testing/testing.go:1334 +0xdf
created by time.goFunc
    /usr/local/go/src/time/sleep.go:169 +0x44
```

--- a/Makefile
+++ b/Makefile
@@ -66,10 +66,10 @@ SUBTESTS :=
 LINTTIMEOUT := 20m
 
 ## Test timeout to use for regular tests.
-TESTTIMEOUT := 12m
+TESTTIMEOUT := 20m
 
 ## Test timeout to use for race tests.
-RACETIMEOUT := 25m
+RACETIMEOUT := 30m
 
 ## Test timeout to use for acceptance tests.
 ACCEPTANCETIMEOUT := 30m

